File: surf-api-paper/surf-api-paper-plugin-test/src/main/java/dev/slne/surf/api/paper/test/command/subcommands/reflection/Reflection.java
package dev.slne.surf.api.paper.test.command.subcommands.reflection;


import dev.slne.surf.api.core.reflection.SurfReflection;

public class Reflection {

    public static final SpigotRestartCommandProxy RESTART_COMMAND;

    static {
        final SurfReflection reflection = SurfReflection.Companion;

        RESTART_COMMAND = reflection.createProxy(SpigotRestartCommandProxy.class);
    }
}

File: surf-api-paper/surf-api-paper-server/src/main/kotlin/dev/slne/surf/api/paper/server/reflection/Reflection.kt
package dev.slne.surf.api.paper.server.reflection

import dev.slne.surf.api.core.reflection.SurfReflection
import dev.slne.surf.api.core.reflection.createProxy

object Reflection {
    val JAVA_PLUGIN_PROXY: JavaPluginProxy = SurfReflection.createProxy<JavaPluginProxy>()
}
